app/react/portainer/environments/ListView/columns/type.tsx

```diff
@@ -16,12 +16,13 @@ export const type = columnHelper.accessor('Type', {
   cell: Cell,
 });
 
-function Cell({ getValue }: CellContext<Environment, EnvironmentType>) {
+function Cell({ getValue, row }: CellContext<Environment, EnvironmentType>) {
   const type = getValue();
+  const containerEngine = row.original.ContainerEngine;
```

Comment:
I tested the accessor with the search bar for tables, and objects returned from the accessor (instead of strings) don't seem to work with search functionality. I'll stick with grabbing the row, and use the platformName (`getPlatformTypeName(row.Type)`) in the accessor

Reply:
updated